server/api_grammar.hpp

```diff
@@ -62,6 +63,14 @@ template <typename Iterator, class HandlerT> struct APIGrammar : qi::grammar<Ite
                     qi::double_)[boost::bind(&HandlerT::addCoordinate, handler, ::_1)];
         hint = (-qi::lit('&')) >> qi::lit("hint") >> '=' >>
                stringwithDot[boost::bind(&HandlerT::addHint, handler, ::_1)];
+        timestamp = (-qi::lit('&')) >> qi::lit("t") >> '=' >>
```

Comment:
timestamp (```t```) behaves exactly like ```hint``` or ```u```. To fix the fact that you can specify e.g. ```classify``` more than once, we would need to change the query grammar in line 43.
Though currently all parameters behave that way.